.github/workflows/_release.yml

```diff
@@ -152,6 +152,10 @@ jobs:
             --extra-index-url https://test.pypi.org/simple/ \
             "$PKG_NAME==$VERSION"
 
+      - name: Run unit tests
```

Comment:
Won't this need some of the poetry test deps?

Reply:
those already installed in L140